apitools/base/py/credentials_lib.py

```diff
@@ -13,7 +14,9 @@
 import oauth2client.locked_file
 import oauth2client.multistore_file
 import oauth2client.service_account
-import oauth2client.tools  # for flag declarations
+# The `tools` import is rarely needed, but takes a long time.
```

Comment:
@craigcitro I ported this comment from the actual changed section below. It seems to be out-of-date (i.e. the slow import is just `oauth2client.crypt` or anything that imports it) but I left it in just in case.